src/main/scala/no/ndla/oembedproxy/model/OEmbedProvider.scala

```diff
@@ -28,7 +28,7 @@ case class OEmbedProvider(providerName: String,
   }
 
   private def _requestUrl(url: String, maxWidth: Option[String], maxHeight: Option[String]): String = {
-    endpoints.find(_.url.isDefined) match {
+    endpoints.find(_.supports(url)).find(e => e.url.isDefined) match {
```

Comment:
Sikkert usannsynelig, men denne _kan_ vel i teorien feile dersom `supports(url)` blir true, men endepunktet som blir funnet ikke har `url`?
Kanskje bytte find'ene med `collectFirst` eller noe lignende?
```scala
private def _requestUrl(url: String, maxWidth: Option[String], maxHeight: Option[String]): String = {
  endpoints.collectFirst { case e if e.supports(url) && e.url.isDefined => e } match {
    case Some(OEmbedEndpoint(_, Some(endpointUrl), _, _, mandatoryQueryParams)) =>
      val embedUrl = endpointUrl.replace("{format}", "json") // Some providers have {format} instead of ?format=
      val width = maxWidth.map(("maxwidth", _)).toList
      val height = maxHeight.map(("maxheight", _)).toList
      val params = List(("url", url), ("format", "json")) ++ mandatoryQueryParams ++ width ++ height
      Url.parse(embedUrl).addParams(params)
    case _ => throw new RuntimeException(s"The provider '$providerName' has no embed-url available")
  }
}
```

Reply:
Lite sannsynlig ja, men endrer likevel.